tensorflow_serving/model_servers/server_core.h

```diff
@@ -117,6 +117,9 @@ class ServerCore : public Manager {
     // Time interval between file-system polls, in seconds.
     int32 file_system_poll_wait_seconds = 30;
 
+    // Timeout for waiting on a servale to reach state.
+    int64 servable_state_monitor_timeout = -1;
```

Comment:
nits:
- name suffix: timeout_micros (for consistency with other parts of tensorflow-serving codebase)
- don't use -1 to denote "no timeout". instead make the type optional<uint64>